src/app/file-manager/file-explorer/file-explorer.component.ts

```diff
@@ -97,6 +105,7 @@ export class FileExplorerComponent implements OnInit {
   deleteNode(node: TreeNode) {
     node.parent.children = node.parent.children.filter(n => n.data !== node.data);
     this.countFiles();
+    this.sumSize();
```

Comment:
Looking at this change, it looks like `deleteNode()` is broken in that it could leave a node selected that's been deleted. Since we don't support delete this isn't the worst problem, but I think we should either fix deleteDelete or remove it until we have a use for it.